docs/core/compatibility/syslib0001.md

```diff
@@ -0,0 +1,46 @@
+---
+title: SYSLIB0001 warning
+description: Learn about the obsoletions that generate compile-time warning SYSLIB0001.
+ms.topic: reference
+ms.date: 10/20/2020
+---
+# SYSLIB0001: The UTF-7 encoding is insecure
```

Comment:
I like the way you've include the warning code, the warning text and the key phrases in the title, H1, and descriptions. That's really well done. (Applies to all these files)

Reply:
Thanks Bill. Actually I changed the headings a bit, mostly to shorten them. Do you think they should match the warning text exactly? Those are listed [here](https://github.com/dotnet/runtime/blob/master/src/libraries/Common/src/System/Obsoletions.cs).